### DISCUSSION TITLE
Service not accessible without router

### DISCUSSION THREAD

**Discussion by nevik25** [zwave-js/zwave-js-ui#2831] [Q&A]
Hi all,
I installed zwavejsui on a docker VM (Proxmox and Debian 11 OS with docker), I've got portainer too on docker, I don't know why when I disconnect my router I can't access to the web UI, it's like the container isn't present or up... MQTT didn't received any information from zwavejsui without router and works fine with. I tried different things and I don't know. When I disconnected my router I have access to portainer wich is in the same docker system...
If someone can help me please..

Thanks.

**Comment by robertsLando** [comment]
I don't understand what you mean here... What's your LAN setup? When you say "I can't access the web UI" you mean from external or from your local LAN? How is your local LAN configured?

**Comment by nevik25** [comment]
I can't acces from my local LAN, my LAN is a classic configuration with my router connected to a NUC with wire on my NUC I've got promox with a VM with Debian, mosquitto and zwave-js

**Reply by robertsLando** [reply]
If you disconnect the router how can you reach the pc? Direct connection with an ethernet cable?

**Reply by nevik25** [reply]
I put a switch between to get a direct connection with my laptop

**Reply by robertsLando** [reply]
is the ip address static or is it set in dhcp?

**Reply by nevik25** [reply]
static

**Reply by robertsLando** [reply]
can you ping the device at least?

**Reply by nevik25** [reply]
Yes, and I accessed to mqtt and portainer on the same ip with a diffrent port

**Reply by robertsLando** [reply]
This is a network issue in your setup, sincerly I dunno what it could be as I never used portainer. @kpine @blhoward2 any clue?

**Reply by blhoward2** [reply]
You’re not exposing a port in the stack most likely if it’s not working at all. I’m confused what you mean by turning off the router. That shouldn’t impact you using the IP address directly, but it could if you have some weird routing setup (on either the NUC or your laptop). Or if you’re using a local mdns (e.g. homeassistant.lan) instead of the IP.
